### DISCUSSION TITLE
Seurat Wrappers

### DISCUSSION THREAD

**Discussion by coolmak32** [satijalab/seurat#5274] [General]
Hi, I am having trouble downloading wrappers from GitHub. 
"Error in utils::download.file(url, path, method = method, quiet = quiet,  : 
  download from 'https://api.github.com/repos/satijalab/seurat-wrappers/tarball/HEAD' failed"

Can anyone please help me with this?

Thank you

**Comment by samuel-marsh** [comment]
Hi,

can you please post the full code you are running to get errors?

thanks,
Sam
